package.json

```diff
@@ -8,40 +8,45 @@
   "license": "UNLICENSED",
   "scripts": {
     "start": "sanity start",
-    "test": "sanity check"
+    "test": "sanity check",
+    "postinstall": "patch-package"
   },
   "keywords": [
     "sanity"
   ],
   "dependencies": {
     "@lifeomic/attempt": "^3.0.0",
-    "@newfrontdoor/audio-player": "^0.1.2",
-    "@newfrontdoor/bible": "0.2.3",
-    "@sanity/base": "^0.144.1",
-    "@sanity/color-input": "^0.144.0",
-    "@sanity/components": "^0.144.2",
-    "@sanity/core": "^0.144.0",
-    "@sanity/dashboard": "^0.144.1",
-    "@sanity/default-layout": "^0.144.2",
-    "@sanity/default-login": "^0.144.0",
-    "@sanity/desk-tool": "^0.144.2",
-    "@sanity/google-maps-input": "^0.144.2",
-    "@sanity/vision": "^0.144.2",
+    "@newfrontdoor/audio-player": "^0.3.2",
+    "@newfrontdoor/bible": "^1.0.0",
+    "@newfrontdoor/s3-file-upload": "^0.4.3",
+    "@sanity/base": "^2.1.3",
+    "@sanity/color-input": "^2.1.0",
+    "@sanity/components": "^2.1.0",
+    "@sanity/core": "^2.1.2",
+    "@sanity/dashboard": "^2.1.0",
+    "@sanity/default-layout": "^2.1.3",
+    "@sanity/default-login": "^2.1.0",
+    "@sanity/desk-tool": "^2.1.3",
+    "@sanity/google-maps-input": "^2.1.3",
+    "@sanity/vision": "^2.1.0",
     "aws-sdk": "^2.550.0",
     "prop-types": "^15.7.2",
     "react": "^16.2",
     "react-dom": "^16.2",
     "react-dropzone": "^10.1.8",
     "react-icons": "^3.7.0",
     "react-spinners": "^0.6.1",
-    "sanity-plugin-dashboard-widget-feed": "0.1.3"
+    "sanity-plugin-dashboard-widget-feed": "0.1.3",
+    "speakingurl": "^14.0.1",
+    "theme-ui": "^0.3.5"
   },
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "eslint": "^6.3.0",
     "eslint-config-xo-react": "^0.20.0",
     "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-react-hooks": "^2.0.1",
+    "patch-package": "^6.2.2",
```

Comment:
This works, for now, but will be a little painful to update.. Hopefully sanity fixes the issue soon..

Reply:
👍